app/models/onboarding_message.rb

```diff
@@ -0,0 +1,38 @@
+class OnboardingMessage < ActiveRecord::Base
+  acts_as_paranoid
+  acts_as_taggable
+
+  has_many :sent_messages, as: :message, dependent: :destroy
+
+  validates :body, presence: true
+  validates :days_after_start, presence: true
+  validates :tag_list, presence: true
+  validates :time_of_day, presence: true
+  validates :title, presence: true
+
+  def self.active
+    where("end_date IS NULL OR end_date > ?", Date.today)
+  end
+
+  def self.date_time_ordering
+    order(:days_after_start, :time_of_day)
+  end
+
+  def self.filter(params)
```

Comment:
since this is exactly the same here and in `QuarterlyMessage` -- thoughts on putting it in a shared concern? http://api.rubyonrails.org/classes/ActiveSupport/Concern.html